frontend/server/src/Controllers/Course.php

```diff
@@ -1408,7 +1408,8 @@ public static function apiAddProblem(\OmegaUp\Request $r): array {
                 'problemsetNotFound'
             );
         }
-
+        
+        $cntProblems = \OmegaUp\DAO\ProblemsetProblems::countProblemsetProblems($problemset);
```

Comment:
--Could you use the whole word:
```suggestion
        $countProblems = \OmegaUp\DAO\ProblemsetProblems::countProblemsetProblems($problemset);
```
?